webanno/webanno
# #1452 - Option to create annotation links without role label

**What's in the PR**
- Add option to enable/disable role labels for link features
- If role labels are disabled the UI label of the target span is shown instead

**How to test manually**
* Have two linked layers
* In the feature options for the link feature disable role labels
* See in the editor view that there is no possibility to enter a role label

**Automatic testing**

**Documentation**
